electron/ironfish/AccountManager.ts

```diff
@@ -131,6 +123,13 @@ class AccountManager
     }
   }
 
+  async create(name: string): Promise<WalletAccount> {
+    return this.node.wallet.createAccount(name).then(account => {
+      this.onAccountCountChange()
+      return account.serialize()
+    })
```

Comment:
suggestion (non-blocking): IMO, mixing async function and then/catch callbacks is a little odd since a big benefit of async functions is keeping things flat. In this case, this could be written as:

```ts
  async create(name: string): Promise<WalletAccount> {
    const account = await this.node.wallet.createAccount(name)
    this.onAccountCountChange()
    return account.serialize()
  }
```

Reply:
Yea makes sense and I agree, this is just revert to previous state.